planetary-social/nos
# Clean up NIP-96 and NIP-98 code

## Issues covered
#1223

## Description
Just cleaned up the code as described in the ticket. Specifically:

- deleted the old file upload code (that uses v2 of the nostr.build API)
- if we don't have the API URL before a user tries to upload an image, fetch the API URL from the nip96.json endpoint
- don't get server info when the app is launched to minimize processing and improve performance at app startup
- deleted refreshServerInfo() since we're no longer using it

## How to test
1. Post
2. Select a photo to upload
3. Make sure it still works

This is purely refactoring, so nothing should change in the UI.
